rgw/v2/utils/utils.py

```diff
@@ -659,7 +659,9 @@ def disable_async_data_notifications():
     by setting rgw_data_notify_interval_msec=0. This will test at level 20,
     the rgw log does not show 'notifying datalog change' entries
     """
-    rgw_log_path = "sudo ls -t /var/log/ceph/*/ceph-client.rgw*"
+    set_log = "ceph config set global log_to_file true"
+    out = exec_shell_cmd(set_log)
```

Comment:
@viduship Could you please add above 2 line in below block
https://github.com/red-hat-storage/ceph-qe-scripts/blob/00b087bde72ca811e00b5e2eb4c7ca8b3c994aa7/rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py#L178